src/lattices.jl

```diff
@@ -355,7 +357,12 @@ function modulate(flat::AbstractFourierLattice{D},
     if !isnothing(expon) && !iszero(expon) 
         @inbounds for i in 2:length(orbits) # leaves the constant term untouched 
                                             # (there will _always_ be a constant term)...
-            modulation[i] /= (norm(first(orbits[i])))^expon
+            n = if isnothing(Gs)
+                norm(first(orbits[i]))^expon
```

Comment:
Ouch; we doubled up on `^expon` here. Fixed in https://github.com/thchr/Crystalline.jl/commit/387362ae5d5c2f24be36cf83e5897ad2ff1cb56c :)